packages/react-composites/src/composites/common/CustomDataModelUtils.tsx

```diff
@@ -61,10 +54,10 @@ const shouldUpdate = (
  * @private
  */
 const avatarDeepEqual = (currentData?: AvatarPersonaData, newData?: AvatarPersonaData): boolean => {
-  return currentData?.text !== newData?.text &&
-    currentData?.imageUrl !== newData?.imageUrl &&
-    currentData?.initialsColor !== newData?.initialsColor &&
-    currentData?.imageInitials !== newData?.imageInitials &&
+  return currentData?.text !== newData?.text ||
```

Comment:
Whoops!

Please add a couple simple unit-tests for this function, now that we've bugged it once already :P

Reply:
will do!